src/Grphp/Client/Interceptors/Base.php

```diff
@@ -53,7 +53,7 @@ public function __construct(array $options = [])
      * @param callable $callback
      * @return Response
      */
-    abstract public function call(callable $callback);
+    abstract public function call(callable $callback): Response;
```

Comment:
This was my main intention: ensuring consistent return type, because we already expecting it in the caller `Client` class: https://github.com/bigcommerce/grphp/blob/e46a743ef33789c4766c35e50f7d9a51f4ecec67/src/Grphp/Client.php#L76